src/lib/pages/contract-details/components/contract-state/index.tsx

```diff
@@ -23,14 +22,15 @@ interface ContractStatesProps {
 }
 
 export const ContractStates = ({ contractAddress }: ContractStatesProps) => {
+  const numStatesToLoad = 100;
```

Comment:
move out of the function make it uppercase `NUM_STATES_TO_LOAD` or move to the `constants.ts`